docs/entitlements-flow.md

```diff
@@ -87,6 +87,24 @@ subscriptions.setOnEntitlementsResponse(function(entitlementsPromise) {
   });
 })
 ```
+## Validate entitlements
+To verify the entitlements from Google are valid and authorised, check the validity of the JWT representation of the entitlements.
+
+The following registered claims are used:
+
+| Claim | Value | Description |
+| ----- | ----- | ----------- |
+| `exp` | [NumericDate](https://tools.ietf.org/html/rfc7519#page-6) (e.g. `1543596936` ) | When the token expires |
+| `iat` | [NumericDate](https://tools.ietf.org/html/rfc7519#page-6) (e.g. `1543595136` ) | When the token was issued |
```

Comment:
Use https://tools.ietf.org/html/rfc7519#section-4.1.6

Reply:
Kept the link to the NumericDate specification but added a link directly to the spec for the different claims.